src/Sarif/Schemata/sarif-schema.json

```diff
@@ -105,6 +105,18 @@
       "type": "object",
       "properties": {
 
+        "index": {
+          "description": "The index within the run artifacts array of the artifact object.",
+          "type": "integer",
+          "default": -1,
+          "minimum": -1
+        },
```

Comment:
we don't need this index as it is not useful within the artifacts table itself (only useful if the artifact would be inlined elsewhere, as we actually do not plan to do, see my other comment).

the description property should remain, though.

Reply:
please more 'index' and then merge.

---
In reply to: [275451263](https://github.com/Microsoft/sarif-sdk/pull/1424#discussion_r275451263) [](ancestors = 275451263)

Reply:
oops! i missed this change, will update!

---
In reply to: [275541609](https://github.com/Microsoft/sarif-sdk/pull/1424#discussion_r275541609) [](ancestors = 275541609,275451263)